### PR TITLE
Created /config/api.js

### PR DESCRIPTION
**Pull request by swewainehtet**
Renamed `apiEndpoint` to `SERVER_IP`
Created `/config/api.js` for future API related exports

### CODE REVIEW

**Comment by swewainehtet** [comment]
Also fixed hardcoded IPs in current axios calls

File: Screens/ForgotPasswordScreen.js [hunk 1]
```diff
@@ -87,154 +81,157 @@ const ForgotPasswordScreen = ({navigation}) => {
...
+        fontSize: 14,
+        color: "#888",
     },
 
-    bottomText:{
-      marginTop: 10,
-      marginBottom: 10,
-      color: '#6E8641', fontWeight: '700'
-    }
-  });
```
Comment (jason2020) [inline Screens/ForgotPasswordScreen.js:240]: Nit: missing newline

File: Screens/ForgotPasswordScreen.js
Comment (jason2020) [inline Screens/ForgotPasswordScreen.js:1]: Sorry, what is this change? Looks good to me I think, but could you document the change in the PR description, ty!

Comment (jason2020) [reply Screens/ForgotPasswordScreen.js:1]: Ideally this change seems like it should be in a separate PR, but this is fine

Comment (jason2020) [reply Screens/ForgotPasswordScreen.js:1]: Oh, I see this is from the linting, okay LGTM. Will fix that to avoid mega diffs in the future.

**Review by jason2020 — APPROVED** [review]
LGTM

**Pull request merged**